client/src/ER/ER.js

```diff
@@ -59,6 +59,14 @@ export default function ER() {
   useEffect(() => {
     if (lastMessage?.data) {
       const data = JSON.parse(lastMessage.data);
+      if (hospitalUser?.hospital.id && window.env.REACT_APP_PILOT_SHOW_ALL_RINGDOWNS === 'true') {
+        data.ringdowns = data.ringdowns.map((r) => {
+          if (r.hospital.id !== hospitalUser.hospital.id) {
+            r.patient.chiefComplaintDescription = `TEST ONLY, NO ARRIVAL. ${r.patient.chiefComplaintDescription}`;
+          }
+          return r;
+        });
+      }
```

Comment:
This is a quick client-side change to show the TEST ONLY, NO ARRIVAL prefix on the ringdown if it wasn't intended for the current ER.